src/OpenTelemetry/Trace/Export/Internal/NoopActivityProcessor.cs

```diff
@@ -32,7 +32,11 @@ public override void OnEnd(Activity activity)
 
         public override Task ShutdownAsync(CancellationToken cancellationToken)
         {
+#if NET452
+            return Task.FromResult(0);
```

Comment:
This one and the ones below are for shutdowns but if it appears in some method called frequently during the lifetime of the process it should be cached in a static.

Reply:
Even though I agree, I'd say it's not worth in optimizing for 4.5.2 over maintainability

Reply:
> I'd say it's not worth in optimizing for 4.5.2 over maintainability

Agreed, just saying that we should pay attention when using non-cached `Task.FromResult`.